src/components/alert-notifications/notification-item.vue

```diff
@@ -32,11 +32,10 @@ function onEnter(el: Element, done: () => void) {
   >
     <div
       v-if="show"
+      v-html="props.notification.message"
```

Comment:
In other places, we already use `v-dompurify-html`. I think this would be safer, as it sanitizes the html.

Reply:
Good point! updated to `v-dompurify-html`